jwpincus/punstartr
# [wip] josh_basic_charts

#### What does  this PR do?

adds the controller/endpoints/page required to see some basic charts on `/statistics`

#### Where should the reviewer start?

All the goodies are in:

- `statistics_controller.rb`
- `views/statistics/index.html.erb`


#### How should this be manually tested?

no tests, but it makes lots of AR calls that make the server work hard. Could be cached, perhaps. (Watch the logs when visiting `/statistics` and you'll see what I mean)

#### Any background context you want to provide?

Implemented Highcharts, used:
https://www.sitepoint.com/make-easy-graphs-and-charts-on-rails-with-chartkick/
https://www.highcharts.com/

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)

looks like this:

![charts](https://cl.ly/330y2q3S3l2V/Cursor_and_Puntstartr.jpg)

#### Questions:
  - Do Migrations Need to be ran? 
no
  - Do Environment Variables need to be set? 
no
  - Any other deploy steps? 
no
